lib/govuk_content_schemas/allowed_document_types.yml

```diff
@@ -174,6 +174,7 @@
 - welsh_language_scheme
 - working_group
 - world_location
+- world_location_news
 - world_location_news_article
```

Comment:
Sorry, I've been out of the loop on this for a little while and forgotten almost everything. What's the difference between this new format and the world_location_news_article below it?

Reply:
cracking, thanks